docs/reference-guides/data/data-core-block-editor.md

```diff
@@ -1089,6 +1089,18 @@ _Returns_
 
 -   `boolean`: Is navigation mode enabled.
 
+### isRemovalPromptDisplayed
```

Comment:
Perhaps the new actions and selectors could be made private for now, at least until we create an API for configuring prompts on any block or editor.

Reply:
Yes, let's make everything private.